README.md

```diff
@@ -14,12 +14,12 @@ or run `make` in this directory for a list of helpful commands.
 Maintainers:
 
 * [Gabriel Devenyi][devenyi_gabriel]
-* [Ashwin Srinath][srinath_ashwin]
 * [Colin Morris][colin_morris]
 * [Will Pitchers][will_pitchers]
+* [Gerard Capes][[gerard_capes]
```

Comment:
Typo: Double open square bracket :(

Reply:
And there's a failure of my highligher :-/

Reply:
I really need to wean myself off of nano....